### PR TITLE
Updating configuration scripts and readme for bundle

### PR DESCRIPTION
**Pull request by ericspod**
Signed-off-by: Eric Kerfoot <eric.kerfoot@kcl.ac.uk>

### Description
Updates to the scripts for 1.0 and readme file. Moved common definitions to a common file used for both inference and training.

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Comment by ericspod** [comment]
The issue appears to be the fact that two config files are needed when running training or inference. There's a `common.json` file that is needed by `train.json` or `inference.json`.

File: models/valve_landmarks/configs/common.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (yiheng-wang-nv) [inline models/valve_landmarks/configs/common.json]: Hi @ericspod , in MONAI label, deploy and FL, it is required to have some necessary keys within the train.json and inference.json, for example: `network_def`, `bundle_root` (like in: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/monaibundle/lib/trainers/bundle.py#L30). You can also refer to: https://github.com/Project-MONAI/model-zoo/issues/52 for more details.
These requirements are for bundles in model zoo, thus could we go back to put these common components back? Or do you have any other suggestions? Thanks!

**Comment by ericspod** [comment]
I've just changed the scripts back to how they were.

File: models/valve_landmarks/configs/common.json (hunk 1 above)
Comment (wyli) [reply models/valve_landmarks/configs/common.json]: I think maybe we should allow for different versions of the same bundle item, with some versions support the preferred keywords and the others less restrictive?

**Review by wyli — APPROVED** [review]

File: models/valve_landmarks/configs/common.json (hunk 1 above)
Comment (ericspod) [reply models/valve_landmarks/configs/common.json]: It wasn't an issue of keywords being absent, the issue was that multiple config files were needed together, ie. this was in the command line: `-config_file "['configs/inference.json', 'configs/common.json']"` The `common.json` file has common definitions, this eliminated duplication but had the side effect of not testing correctly. I'll propose a feature later but we need some way of including one config file in another with a `_include_` tag or something like that.

Comment (Nic-Ma) [reply models/valve_landmarks/configs/common.json]: Hi @ericspod ,

Our `%` syntax supports macro replacement, so actually we can use `"include": "%my/another/config.json"`?

Thanks.

Comment (ericspod) [reply models/valve_landmarks/configs/common.json]: Would it recognise that as a filename and expand all the items in that file even without any # syntax? Is the path relative to the current file? If yes that would be the solution though we should discuss it in more detail in the docs.

Comment (Nic-Ma) [reply models/valve_landmarks/configs/common.json]: @ericspod yes, but the path may be same as the CLI. For more details, please take a look at: https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/config_parser.py#L327

Thanks.

Comment (ericspod) [reply models/valve_landmarks/configs/common.json]: Thanks Nic, this will work but one would have to refer to components by the key for the macro, eg. "$@include.network_def.to(@include.device)". If might be easier to have a `_include_` tag that would copy values into the current namespace and also give feedback if a circular import is encountered.

Comment (Nic-Ma) [reply models/valve_landmarks/configs/common.json]: Interesting, so it's something like `macro replacement` not only on values with `%`, but also on the keys.
CC @wyli @yiheng-wang-nv for more discussion.

Thanks.
